lib/v2/screens/dashboard/components/transactions_list/interactor/viewmodels/member_state.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:equatable/equatable.dart';
+import 'package:seeds/v2/datasource/remote/model/member_model.dart';
+import 'package:seeds/v2/domain-shared/page_state.dart';
+
+class MemberState extends Equatable {
+  final PageState pageState;
+  final String accountName;
+  final MemberModel? member;
+
+  String get displayName => (member != null && member!.nickname != "") ? member!.nickname : accountName;
```

Comment:
member!.nickname != "" 
member!.nickname.isEmpty

Reply:
Honestly != "" makes it more readable for me, rather than having to parse "isNotEmpty" vs. "isEmpty"?!

But OK?!
\